src/Microsoft.PackageManagement.ArchiverProviders/project.json

```diff
@@ -9,9 +9,25 @@
     "configurations": {
         "Linux": {
             "buildOptions": {
-                "define": ["UNIX"]
+                "define": [ "UNIX" ],
+                "debugType": "portable"
+            }
+        },
+        "Debug": {
+                "buildOptions": {
+                    "debugType": "portable"
+                }
+            },
+        "Release": {
+                "buildOptions": {
+                    "debugType": "portable"
+                }
+            },
+        "CodeCoverage": {
+                "buildOptions": {
+                    "debugType": "full"
+                }
```

Comment:
The changes in project.json files are respecting the existing indentation pattern. Could you please update those changes to use the same existing indentation patterns?